### ISSUE TITLE
EXCPT opcode does not properly unwind the stack.

### ISSUE THREAD

**Issue by gitlarryf** [ghewgill/neon-lang#189]
The following test demonstrates how the wrong frame is active during the exception handler, because the stack is not properly unwound when an exception is thrown.

```
EXCEPTION TestException

FUNCTION func1()
    VAR s: String := ""
    s := s % Ensure the variable is used
    RAISE TestException
END FUNCTION

FUNCTION func2()
    VAR s2: Number := 2
    s2 := s2 % Ensure the variable is used
    TRY
        func1()
    TRAP TestException DO
        print("\(s2)") % accesses wrong Frame / wrong local variable
    END TRY
END FUNCTION

func2()

```
This only works with functions, since global variables are not handled with a frame.

**Comment by ghewgill** [comment]
The `exception-unwind` branch has a bit of preliminary exploratory work on this that doesn't fix the full problem yet.

**Comment by ghewgill** [comment]
Fixed in 49c489ec94c93e04e6a2fa3df36f046b44526545.
